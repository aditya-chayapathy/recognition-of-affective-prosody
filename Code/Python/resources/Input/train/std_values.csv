artistfluffy.mat, 130.1065
cleanbedroomcleanbedroom.mat, 52.1885
cleanbedroomelephant.mat, 47.8992
cleanbedroomhighbuildings.mat, 180.0630
cleanbedroomrelaxed.mat, 64.5627
dogartist.mat, 54.4542
dogelephant.mat, 51.5128
doghighbuildings.mat, 47.1075
elephanthighbuildings.mat, 49.2928
fluffyelephant.mat, 128.7595
fluffyfluffy.mat, 65.8416
fluffynorthpole.mat, 41.9371
hamburgerhighbuildings.mat, 43.0408
hamburgernorthpole.mat, 46.6109
highbuildingsfluffy.mat, 90.8101
highbuildingshamburger.mat, 23.5644
highbuildingshighbuildings.mat, 44.9506
highbuildingslittlegirl.mat, 53.6051
littlegirlcleanbedroom.mat, 45.8135
littlegirldog.mat, 45.6930
littlegirlfluffy.mat, 48.6274
littlegirlhighbuildings.mat, 252.7661
littlegirllittlegirl.mat, 86.0550
littlegirlnorthpole.mat, 17.1196
northpoledog.mat, 51.8365
relaxedcleanbedroom.mat, 91.2595
relaxeddog.mat, 17.1174
relaxedhamburger.mat, 140.5032
relaxedrelaxed.mat, 58.9226
