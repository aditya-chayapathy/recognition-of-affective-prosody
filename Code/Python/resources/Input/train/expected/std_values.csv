BirdMonkey.mat, 18.5946
CatCat.mat, 19.3630
CatDog.mat, 21.2902
CatHorse.mat, 22.9138
DogDog.mat, 20.9283
DogFish.mat, 22.1223
ElephantBird.mat, 21.5021
ElephantElephant.mat, 23.4255
ElephantHorse.mat, 29.5196
FishFish.mat, 17.5378
FishPanda.mat, 18.6486
HorseBird.mat, 21.6885
HorseCow.mat, 19.9750
HorseMonkey.mat, 19.8382
MonkeyBird.mat, 22.3871
MonkeyKangaroo.mat, 20.2441
PandaCat.mat, 19.9530
